### PR TITLE
Created button component  Issue #1

### PR DESCRIPTION
**Pull request by ShivangiRai1310**
#1 Created button component, added the default prop values.
Tested the button component on App.js, have attached the pictures for the same.

Kindly review the PR and let me know anything that is to be changed.

Screenshot (240)

Screenshot (237)

### CODE REVIEW

File: src/components/common/button/button.js [hunk 1]
```diff
@@ -1,12 +1,46 @@
 import React from 'react'
 import './button.css'
 
-const Button = () => {
+const Button = (props) => {
+
+    console.log(props);
+    let sec;
+    if( props.variant === "secondary")
```
Comment (smilegupta) [inline src/components/common/button/button.js:8]: can u remove this console.log statement

File: src/components/common/button/button.js [hunk 2]
```diff
@@ -1,12 +1,46 @@
...
+
+
+<Button label="Click Me" />
+<Button label="Text to add" 
+        type="button/submit/reset" 
+        variant="primary/secondary" 
+        isLarge={true/false} 
+        disabled={true/false} 
+        />
+*/
```
Comment (smilegupta) [inline src/components/common/button/button.js:46]: can u please remove the commented code

**Review by smilegupta — CHANGES_REQUESTED** [review]
Can you remove some extra commented code written in the  files.
Also if possible can you prettify the code once using prettifier ?

**Comment by ShivangiRai1310** [comment]
Done the changes:
1. Removed additional comments and console.log
2. Used Prettifier

**Comment by ShivangiRai1310** [comment]
Hii, @smilegupta could you please review the changes and merge it if there is no issue with the code.

**Review by smilegupta — APPROVED** [review]

**Comment by smilegupta** [comment]
Greetings from Build With Girlscript Bangalore😃

Dear participant,

We have gone through your pull request made for the issue #18 
 
Congratulations🎊 🎉 on getting your pull request successfully merged into our application **Build with GSBlr-Raahat**. 

We will soon be distributing your valuable badge(s)🥇🥈🥉🎖️ corresponding to the pull requests that are merged. Please fill the [participation form](https://tinyurl.com/buildwithgsblr) 📃 to receive your badges, if not already done.

Also, [RSVP here](https://tinyurl.com/gsblr-hacktoberfest2020) here, if you haven't yet,  to get a chance to win a limited-edition Hacktoberfest 👕 T-shirt or a 🎍plant.

Learning is a never-ending process. 📈  We will be coming up with *new issues*, stay tuned. Meanwhile you can also check out our other [Code with GirlScript Bangalore Repository](https://github.com/girlscript-blr/code-with-girlscript-bangalore) to try some problem solving and earn exciting badges.

Thanks for your contribution! Happy coding😃

**Comment by girlscript-blr** [comment]
Hi @ShivangiRai1310 

Please fill the [participation form](https://tinyurl.com/buildwithgsblr) 📃 to receive your badges.

Thanks

**Pull request merged**